package.json

```diff
@@ -995,6 +1149,42 @@
           "type": "boolean",
           "default": false,
           "description": "Remove hidden items when clearing workspace."
+        },
+        "r.plot.defaults.colorTheme": {
```

Comment:
Do we really need that many levels of config key?

`r.plot.defaultColorTheme`, `r.plot.defaultPlotPreviewLayout`, `r.plot.resizeTimeout`, `r.plot.refreshTimeout` seem to look good enough.

Reply:
My motivation for nesting config entries this way is to control how they are sorted in the settings UI. Currently, this isn't a big deal, but if we simplify the config keys and later add e.g. `r.plot.resizeOnEventXYZ`, it will be displayed between `r.plot.resizeTimeout` and `r.plot.refreshTimeout`, though it would make more sense to group the timeouts together.

Reply:
OK. I also guessed it is intended for sorting, then it is useful indeed.

BTW, the timeouts seem more like intervals? I saw other extensions use "timeout" to limit the amount of time to wait until success (e.g. connection timeout). "Interval" seems more accurate for the purpose?

Reply:
Makes sense

Reply:
Still using `r.plot.timeouts.*`?

Reply:
Yeah I figured somewhere in the name it should indicate that it's a time-interval (since "interval" is not used uniquely for time delays). But we can also group it as `r.plot.timing.*` etc. if that sounds better, I don't really mind :)

Reply:
`r.plot.timing.*` sounds ok.